templates/discofuse/discofuse-sport/templates.yaml

```diff
@@ -0,0 +1,145 @@
+dataset: discofuse
+subset: discofuse-sport
+templates:
+  03f85406-df19-4bba-9ff7-53e050db6c84: !Template
+    id: 03f85406-df19-4bba-9ff7-53e050db6c84
+    jinja: "{% if coherent_second_sentence==\"\" %}\n\nDecompose the following sentence\
+      \ into two separate sentences:\n\n{{coherent_first_sentence}}\n\n|||\n\n{{incoherent_first_sentence}}\
+      \ {{incoherent_second_sentence}} \n{% endif %}"
+    name: decompose 2
+    reference: ''
+    task_template: false
+  0e00ea8a-dc1b-4b3d-9f6f-13378e6e739d: !Template
+    id: 0e00ea8a-dc1b-4b3d-9f6f-13378e6e739d
+    jinja: "{% if discourse_type != \"PAIR_NONE\" %}\n\nPassage 1: {{incoherent_first_sentence}}\
+      \ {{incoherent_second_sentence}}\n\nPassage 2: {{coherent_first_sentence}} {{coherent_second_sentence}}\n\
+      \nWhich of the following discourse phenomena have been used to turn Passage\
+      \ 1 into Passage 2?\n\n{{\"A: Apposition\"}}\n\n{{\"B: Relative Clauses\"}}\n\
+      \n{{\"C: Cataphora\"}}\n\n{{\"D: Verb Phrase Coordination\"}}\n\n{{\"E: Anaphora\"\
+      }}\n\n{{\"F: Inner Connectives\"}}\n\n{{\"G: Both Inner Connectives and Anaphora\"\
+      }}\n\n{{\"H: Sentence Coordination\"}}\n\n{{\"I: Both Sentence Coordination\
+      \ and Anaphora\"}}\n\n{{\"J: Forward Connectives\"}}\n\n{{\"K: Discourse Connectives\"\
+      }}\n\n{{\"L: Both Discourse Connectives and Anaphora\"}}\n\nAnswer with both\
+      \ the option letter and phenomena name.\n|||\n\n{{\n{\"PAIR_ANAPHORA\": \"E:\
```

Comment:
In most cases, when we have a multiple-choice question, we have been just having the model predict the label (A, B, C, etc) rather than the label + text. Do you mind making this change?